tests/mgbench/client.cpp

```diff
@@ -220,7 +224,111 @@ nlohmann::json LatencyStatistics(std::vector<std::vector<double>> &worker_query_
   return statistics;
 }
 
-void Execute(
+void ExecuteTimeDependentWorkload(
```

Comment:
Added a new function for time depended execution. Most of the code is shared, but results and exiting differ from normal execution. If you whan't, we can stitch everything into same function.